lib/src/components/movement_sensor/movement_sensor.dart

```diff
@@ -10,6 +10,16 @@ class Position {
   Position(this.coordinates, this.altitude);
 }
 
+class Accuracy {
+  Map<String, double> accuracyMap;
+  double hdop;
+  double vdop;
+  int nmeaFix;
+  double compassDegreeError;
+
+  Accuracy(this.accuracyMap, this.hdop, this.vdop, this.nmeaFix, this.compassDegreeError);
+}
```

Comment:
Similar to python, this should be a typealias. You can basically copy the [next line](https://github.com/viamrobotics/viam-flutter-sdk/pull/170/files#diff-9829cc355197d082e394a03aab4e61e2f8e3397abff5c1c4e00f60c67f45debfL13) for example